include/nccl_ofi_tuner.h

```diff
@@ -0,0 +1,91 @@
+#ifndef NCCL_OFI_TUNER_H_
+#define NCCL_OFI_TUNER_H_
+
+#include <linux/limits.h>
+#include <float.h>
+#include "nccl-headers/tuner.h"
+
+/*
+ * TODO: This should not be statically defined. The plugin interface lets us
+ * tune the number of channels as well, but that can come later (once a
+ * proto+algo combination is chosen, we can compute the cost with different
+ * channel count and optimize for it.
+ */
+#define NCCL_OFI_TUNER_NUM_CHANNELS	(8)
```

Comment:
dont u want to make this an array-like so u can set up different values for different machines.       something like struct nccl_ofi_tuner[] { ....}.  and then and access via nccl_ofi_tuner[EC2_P5]

Reply:
I want to move some of these defines into the platform-specific data in a follow-up PR https://github.com/aws/aws-ofi-nccl/blob/master/src/platform-aws.c#L31; which has the same intent.